### DISCUSSION TITLE
似乎套cf走https，流量很拉稀啊，改走http畅通无阻，这怎么得鸟

### DISCUSSION THREAD

**Discussion by ChainZeaxion** [v2fly/v2ray-core#2151] [General]
https
websocket h1.1
grpc h2
连上，光测httping www.google.com，延迟超千，大部分1500ms
还不如裸连过去，好歹延迟大部分在600
带宽正常跑满200Mbps，被鈤的时候2Mbps

http
websocket h1.1
连上，光测httping www.google.com，延迟700，大部分760ms
带宽正常跑满200Mbps，被鈤的时候，损失20-30%左右

裸连https
websocket h1.1
grpc h2
连上，光测httping www.google.com，延迟300，大部分360ms，被鈤的时候延迟700ms
带宽正常跑满200Mbps，被鈤的时候，20-30Mbps，最惨的时候2Mbps

裸连http
websocket h1.1
连上，光测httping www.google.com，延迟200，大部分230，被鈤的时候延迟600ms
带宽正常跑满200Mbps，被鈤的时候，30-50Mbps，最惨的时候2Mbps

裸icmp，延迟150ms，被鈤的时候延迟430ms，tcp丢包60%，udp丢包45%

问题是，走http，不加tls，容易被鈤到禁ip么？？？
vmess aead chacha1305 websocket
vmess aead chacha1305 tls websocket
对比起来只少一个https

**Comment by ChainZeaxion** [comment]
高峰时段，裸连已经被鈤的不要不要了，正常延迟翻一倍，整的狠了，延迟翻两倍
油管只能勉强1080
改套cf走https，跟裸连大差不差
改套cf走http，握槽，油管4K畅通无阻
真就离谱，看起来只要加tls，就容易被鈤啊

**Comment by kulongwangzhi85** [comment]
我都不套的😄
